### PR TITLE
Expand all, collapse all buttons for configure tags, messages, conversation filters

### PR DESCRIPTION
**Pull request by elayabharath**
Closes https://github.com/larksystems/Katikati-Core/issues/722

### CODE REVIEW

File: webapp/lib/app/nook/view.dart [hunk 1]
```diff
@@ -1243,7 +1243,20 @@ class ConversationFilter {
     conversationFilter.append(_tagsContainer);
 
     _tagAccordion = Accordion([]);
-    _tagsMenuWrapper.append(_tagAccordion.renderElement);
+    var accordionActions = DivElement()..classes.add('filter-accordion-actions');
+    var expandButton = buttons.Button(buttons.ButtonType.text, buttonText: "Expand all", onClick: (e) {
+      _tagAccordion.items.forEach((AccordionItem item) { item.expand(); });
```
Comment (marianamarasoiu) [inline webapp/lib/app/nook/view.dart:1248]: nit: 
```suggestion
      _tagAccordion.items.forEach((AccordionItem item) => item.expand());
```
(this is the dart syntax for one liner functions)

File: webapp/lib/app/nook/view.dart [hunk 2]
```diff
@@ -1243,7 +1243,20 @@ class ConversationFilter {
...
+    });
+    var collapseButton = buttons.Button(buttons.ButtonType.text, buttonText: "Collapse all", onClick: (e) {
+      _tagAccordion.items.forEach((AccordionItem item) { item.collapse(); });
```
Comment (marianamarasoiu) [inline webapp/lib/app/nook/view.dart:1251]:
```suggestion
      _tagAccordion.items.forEach((AccordionItem item) => item.collapse());
```

**Review by marianamarasoiu — COMMENTED** [review]
just a quick question - why aren't these buttons part of the `Accordion` component and instead we have to create them locally? feels like it should be a generalisable feature?

**Comment by elayabharath** [comment]
Closing in favour of https://github.com/larksystems/katikati_common_lib/pull/120

**Pull request closed without merging**